### ISSUE TITLE
Publish profile transforms without configuration transforms throws an error

### ISSUE THREAD

**Issue by kernelcoredump** [microsoft/slow-cheetah#196]
Using version 3.2.20 through MSBuild for web applications, the presence of a publish profile transform without a configuration transform throws the following exception:
`
Microsoft.VisualStudio.SlowCheetah.Web.targets(95,5): error MSB4018: System.IO.FileNotFoundException: File not found
`

For example, having both _Example.config_ and _Example.Staging.config_ (where _Staging_ is a publish profile), but without an existing configuration transform _Example.Debug.config_ or _Example.Release.config_.

In the last 2.x.x version (2.5.48), the SC target copies the original file to an intermediate file and performs the transforms on the intermediate file.  https://github.com/microsoft/slow-cheetah/blob/13b1b4cf0fbfa96d40334099ebc01bd8557982c6/SlowCheetah/Build/SlowCheetah.targets#L168

Using the latest (3.2.20) in my use case, the task at https://github.com/microsoft/slow-cheetah/blob/86bdac1732c2a725e3d9d77efa9d5652d70f3b7e/src/Microsoft.VisualStudio.SlowCheetah/Build/Microsoft.VisualStudio.SlowCheetah.Web.targets#L90 is skipped because there isn't a configuration transform.  But the next task at https://github.com/microsoft/slow-cheetah/blob/86bdac1732c2a725e3d9d77efa9d5652d70f3b7e/src/Microsoft.VisualStudio.SlowCheetah/Build/Microsoft.VisualStudio.SlowCheetah.Web.targets#L95 is looking for the file at the location of the _Destination_ from the skipped task.

**Comment by davilimap** [comment]
You're right. The fix here would probably be to add the condition `Exists('%(PublishTransformFile)')` to the current publish profile transform task then add another transform task with the condition `'$(Configuration)'=='' and '$(ScPubProfile)'!=''`
